eng/Publishing.props

```diff
@@ -49,10 +49,12 @@
       <!-- Do not push .nupkg files from Linux and macOS builds. They'll be packed up separately and signed on Windows. -->
       <ItemsToPushToBlobFeed Remove="@(ItemsToPushToBlobFeed)" Condition="'$(OS)' != 'Windows_NT'" />
 
+      <!-- Skip publishing checksums for now - the checksums for the .zip files don't match (https://github.com/dotnet/aspnetcore/issues/18792)
       <ItemsToPushToBlobFeed Include="@(_ChecksumsToPublish)">
```

Comment:
Why not exclude the known bad files from `@(_ChecksumsToPublish)`? Will it take too long to verify them?

Reply:
That was my thought - I figured it wasn't worth the risk & it was better to get this right the first time we did it, rather than pushing half of the stuff